Classes/Projection/Doctrine/AbstractDoctrineProjector.php

```diff
@@ -32,15 +33,32 @@
      * Make sure to call this method as parent when overriding it in a concrete projector.
      *
      * @return void
+     * @throws Exception
      */
     protected function initializeObject()
     {
-        if ($this->readModelClassName === null && substr(get_class($this), -9, 9) === 'Projector') {
+        if ($this->readModelClassName === null) {
+            if (substr(get_class($this), -9, 9) !== 'Projector') {
+                throw new Exception(sprintf('The class name "%s" doesn\'t end in "Projector" so the Read Model class name can\'t be determined automatically. Please set the "readModelClassName" field it in your concrete projector.', get_class($this)), 1476799474);
```

Comment:
This is not directly related, but if we silently skip this case bad things will happen